lms/djangoapps/completion/migrations/0001_initial.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import lms.djangoapps.completion.models
+import django.utils.timezone
+from django.conf import settings
+import model_utils.fields
```

Comment:
I thought we try to import more explicitly, like `AutoCreatedField`, but maybe not.

Reply:
This is the migrations file.  I think it's better to leave that in its autogenerated state.

Reply:
I believe it's excluded from linting for that reason.

Reply:
Agreed.  Sorry.  My brain misfired.